pkg/api/types.go

```diff
@@ -334,6 +441,8 @@ type Capabilities struct {
 type ResourceRequirements struct {
 	// Limits describes the maximum amount of compute resources required.
 	Limits ResourceList `json:"limits,omitempty"`
+	// Requests describes the minimum amount of compute resources required.
+	Requests ResourceList `json:"requests,omitempty"`
```

Comment:
@bgrant0607 is it worth opening the door to this grey-area ?  Perhaps the first field we created should be "Requests" rather than "Limits" ?